src/charm.py

```diff
@@ -223,6 +244,9 @@ def _on_start(self, _):
         """
         self._common_exit_hook()
 
+    def _on_stop(self, _):
+        self.unit.set_workload_version("")
+
```

Comment:
Do we need to set the workload version to "" in all our charms on stop?

Reply:
I think we should.